src/view/shell/desktop/Feeds.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import {View, StyleSheet} from 'react-native'
+import {useNavigationState} from '@react-navigation/native'
+import {AtUri} from '@atproto/api'
+import {observer} from 'mobx-react-lite'
+import {useStores} from 'state/index'
+import {usePalette} from 'lib/hooks/usePalette'
+import {CogIcon} from 'lib/icons'
+import {TextLink} from 'view/com/util/Link'
+import {getCurrentRoute} from 'lib/routes/helpers'
+
+export const DesktopFeeds = observer(function DesktopFeeds() {
+  const store = useStores()
+  const pal = usePalette('default')
+
+  // NOTE
+  // we use this to trigger re-renders on nav changes
+  // the links establish if they're current by looking at the window.location
```

Comment:
where does this happen? generally we don't want components to be reading mutable stuff like `window.location` during render for this reason, the canonical way is `useSyncExternalStore`.

Reply:
Oh I ended up replacing that behavior and forgot to pull this out